python/cudf/cudf/tests/dataframe/methods/test_between_time_at_time.py

```
@@ -0,0 +1,94 @@
+
+def test_at_time_invalid_index():
+    gdf = cudf.DataFrame({"A": [1, 2, 3]})
+    with pytest.raises(TypeError):
+        gdf.at_time("12:00")
+
+
+def test_at_time_invalid_axis():
+    dates = cudf.date_range("2018-04-09", periods=4, freq="12h")
+    gdf = cudf.DataFrame({"A": [1, 2, 3, 4]}, index=dates)
+    with pytest.raises(NotImplementedError):
+        gdf.at_time("12:00", axis=1)
```

Ruff (0.16.3)

[warning] No newline at end of file

Add trailing newline

(W292)